test/forward_merger.test.ts

```diff
@@ -0,0 +1,292 @@
+    mockContextRepo.mockReset();
+    mockContextRepo.mockReturnValue(repoResp);
+    mockConfigGet.mockResolvedValue(
+      makeConfigReponse({
+        forward_merger: true,
+      })
+    );
+  });
+
+  test("mergeForward should not run on push to non-versioned branch", async () => {
+    const context = makePushContext({
+      ref: "refs/heads/unversioned",
+    });
+    const forwardMerger = new ForwardMerger(context, context.payload);
+    const mockGetBranches = jest.fn().mockName("getBranches");
+    forwardMerger.getBranches = mockGetBranches;
+    await forwardMerger.mergeForward();
+
+    expect(mockGetBranches).not.toBeCalled();
+  });
+
+  test("mergeForward should not run when plugin is disabled", async () => {
+    const context = makePushContext({
+      ref: "refs/heads/branch-21.12",
+    });
+    const forwardMerger = new ForwardMerger(context, context.payload);
+    const mockGetBranches = jest.fn().mockName("getBranches");
+    forwardMerger.getBranches = mockGetBranches;
+    const mockPluginIsDisabled = jest
+      .fn()
+      .mockName("pluginIsDisabled")
+      .mockResolvedValue(true);
+    forwardMerger.pluginIsDisabled = mockPluginIsDisabled;
+    await forwardMerger.mergeForward();
+
+    expect(mockPluginIsDisabled).toBeCalled();
+    expect(mockGetBranches).not.toBeCalled();
+  });
+
+
+  test("mergeForward should not open PR on invalid next branch", async () => {
+    const context = makePushContext({
+      ref: "refs/heads/branch-22.02",
+    });
+    const forwardMerger = new ForwardMerger(context, context.payload);
+    forwardMerger.getBranches = jest
+      .fn()
+      .mockName("getBranches")
+      .mockResolvedValue(null);
+    forwardMerger.sortBranches = jest
+      .fn()
+      .mockName("sortBranches")
+      .mockReturnValue(null);
+    const nextBranch = undefined;
+    const mockGetNextBranch = jest
+      .fn()
+      .mockName("getNextBranch")
+      .mockReturnValue(nextBranch);
+    forwardMerger.getNextBranch = mockGetNextBranch;
+    mockCreatePR.mockResolvedValue(null);
+
+    await forwardMerger.mergeForward();
+
+    expect(mockCreatePR).not.toBeCalled();
+  });
+
+
+  test("should comment success on PR if merge is successful", async () => {
+    const context = makePushContext({
+      ref: "refs/heads/branch-21.12",
+    });
+    const forwardMerger = new ForwardMerger(context, context.payload);
+    forwardMerger.getBranches = jest
+      .fn()
+      .mockName("getBranches")
+      .mockResolvedValue(null);
+    forwardMerger.sortBranches = jest
+      .fn()
+      .mockName("sortBranches")
+      .mockReturnValue(null);
+    const nextBranch = {
+      name: "branch-21.10",
+    };
+    forwardMerger.getNextBranch = jest
+      .fn()
+      .mockName("getNextBranch")
+      .mockResolvedValue(nextBranch);
+    const pr = { data: { number: 1, head: { sha: 123456 } } };
+    mockCreatePR.mockResolvedValue(pr)
+    
+    mockMerge.mockResolvedValue({ merged: true })
```

Comment:
```suggestion
    mockMerge.mockResolvedValue(true)
```

the return value of the merge is not used anywhere in our code, so we can simplify this value to just `true` to avoid ambiguity